datadriven/src/line_sparser.rs

```diff
@@ -0,0 +1,121 @@
+use crate::test_data::CmdArg;
+use anyhow::Result;
+use regex::Regex;
+
+// Token
+// (1) argument (no value)
+// (2) argument= (no value)
+// (3) argument=a (single value)
+// (4) argument=a,b,c (multiple value)
+// (5) argument=(a,b,c,...) (multiple value)
+//
+// parse_line parses a line of datadriven input language and returns
+// the parsed command and CmdArgs.
+pub fn parse_line(line: &str, logger: &slog::Logger) -> Result<(String, Vec<CmdArg>)> {
+    let fields = split_directives(line)?;
+    if fields.is_empty() {
+        bail!("empty lines occurs, unexpected.");
```

Comment:
The original implement is returning empty result.

Reply:
But we get empty fields only when there is no command and arguments, which is `nothing` in test cases.
Maybe we just disallow this condition?

Reply:
I suggest to follow the behavior of go.